README.md

```diff
@@ -1 +1,36 @@
-# publish-integration-image-action
\ No newline at end of file
+# publish-integration-image-action
+
+This is a composite action that will publish you image to your Docker Registry.
+
+## Inputs
+
+This action requires uses the following inputs:
+
+| Name               | Type     | Description                                                                        |
+|--------------------|----------|------------------------------------------------------------------------------------|
+| `package-name`     | String   | Name that the image should be published as to Docker Hub as                        |
+| `docker-username`  | String   | Username for the docker account to publish under                                   |
+| `docker-password`  | String   | Password for the docker account to publish under                                   |
+| `push-to-registry` | String   | Determine if the built image should be pushed to registry (Default is 'false')     |
+
+The docker username and password will likely be passed by `secrets`. You must ensure your repo has access to these secret variables
+
+## Example Usage:
+
+```yaml
+jobs:
+  ...
+
+  publish-image:
```

Comment:
I think we'll want to document the usage pattern here to express that this should really only be published on `main`.